duckduckgo/iOS
# add ui tests, add and expand dividers, show total tracker counts

Task/Issue URL: 

https://app.asana.com/0/414235014887631/559535480721833
https://app.asana.com/0/414235014887631/589111725486548
https://app.asana.com/0/414235014887631/494552239039548
https://app.asana.com/0/414235014887631/494552239039549

Tech Design URL: n/a
CC:

**Description**:

* Tidy up: remove programatic setting of line height - all done in storyboards now
* Added UI Tests to be able to generate easy to eyeball view of UI across a range of devices (run `./screenshots.sh` from the root of the iOS project - takes 10-20 minutes)
* Show total trackers detected on overview and score card
* Added job to run UI tests to Jenkins

Going to get Mike to look at the screenshots capture to make sure he's OK with them as well, so just need a bit of test coverage across the app.

**Steps to test this PR**:

1. Visit cnn.com
1. Explore privacy dashboard and ensure looks OK 
1. Try to touch all functionality (Onboarding, About, Bookmarks, Tabs, Browsing, Privacy Dashboard)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
